projects/uitpas/reference/uitpas.json

```diff
@@ -5302,6 +5380,30 @@
             "description": "Postal code of the passholder of this pass",
             "readOnly": true
           },
+          "socialTariff": {
+            "type": "object",
+            "description": "Information about the possible social tariff of the passholder. This object is always present, however a passholder is only entitled to social tariff if the state property has value `ACTIVE`.",
+            "required": [
+              "status"
+            ],
+            "properties": {
+              "status": {
+                "type": "string",
+                "enum": [
+                  "ACTIVE",
+                  "EXPIRED",
+                  "NONE"
+                ],
+                "description": "Status of the social tariff:\n- `ACTIVE`: the passholder is entitled to social tariff\n- `EXPIRED`: the passholder is NOT entitled to social tariff anymore\n- `NONE`: the passholder is NOT entitled to social tariff"
+              },
+              "endDate": {
+                "type": "string",
+                "format": "date-time",
+                "description": "Exact expiration date of the passholder's entitlement to a social tariff. This property is not available when status is `NONE`.",
```

Comment:
Maybe a bit redundant but probably won't hurt to be as clear as possible here (mostly that the date can be in the future or in the past)

Reply:
Good idea to clarify, but it's not completely correct. It would be more correct this way:
```suggestion
                "description": "Exact expiration date of the passholder's entitlement to a social tariff. This property must not be used to determine the social tariff status, because `status` can be `ACTIVE` while the `endDate` is in the past during a 'grace period'. This property is not available when status is `NONE`. ",
```

Reply:
Sounds good!